### ISSUE TITLE
为什么我在kitti 2015的eval metric比论文和文档里的结果低那么多？Why is my eval metric in Kitti 2015 so much lower than the results in the paper and documentation

### ISSUE THREAD

**Issue by CodeworldKD** [XiandaGuo/OpenStereo#186]
![image](https://github.com/user-attachments/assets/826af615-9400-47ff-bdd9-422bb31c5715)
![image](https://github.com/user-attachments/assets/a85cdcc8-8ffa-4df1-beae-9d0c11b19589)

上面这两个是我在light和base模型上的评估结果，下面是文档里的
The above two are my evaluation results on the light and base models, and below is the document

![image](https://github.com/user-attachments/assets/e7124227-9609-4e93-823a-3c8c955aa572)
怎么回事求解答！！

**Comment by CodeworldKD** [comment]
@XiandaGuo 怎么回事啊哥，我的环境有问题吗还是泄露了答案啊，我是直接使用pretrain模型直接调用eval.py跑的

**Comment by XiandaGuo** [comment]
You should use the pre-trained weights on SceneFlow.

**Comment by Alresd** [comment]
> You should use the pre-trained weights on SceneFlow.
您好，为什么我使用了SceneFlow上预训练的模型但是还是达不到文档里面的指标，求解答！

![Image](https://github.com/user-attachments/assets/a9779722-dbcf-437d-9b00-1fbef5fa04c0)
![Image](https://github.com/user-attachments/assets/a0546703-1971-4351-a93a-a90b7b3884dd)
